ts/utils/hooks/bottomSheet.tsx

```diff
@@ -192,123 +144,48 @@ export const useIOBottomSheetModal = ({
     return () => screenReaderChangedSubscription.remove();
   }, []);
 
-  const footerComponent = footer ? (
-    <View style={{ paddingBottom: insets.bottom }}>{footer}</View>
-  ) : null;
-
   const bottomSheet = (
     <BottomSheetModal
       style={styles.bottomSheet}
-      footerComponent={(_: BottomSheetFooterProps) => footerComponent}
-      snapPoints={[...snapPoint]}
+      footerComponent={(props: BottomSheetFooterProps) =>
+        footer ? (
+          <BottomSheetFooter
+            {...props}
+            // bottomInset={insets.bottom}
+            style={{
+              paddingBottom: insets.bottom,
+              backgroundColor: IOColors.white
+            }}
+          >
+            {footer}
+          </BottomSheetFooter>
+        ) : null
+      }
+      enableDynamicSizing={snapPoint ? false : true}
```

Comment:
Consider updating the `enableDynamicSizing` prop to use `!snapPoint` instead of `snapPoint ? false : true`

```suggestion
      enableDynamicSizing={!snapPoint}
```